py3status/modules/thunderbird_calendar.py

```diff
@@ -60,22 +63,18 @@ def get_calendar(self, i3s_output_list, i3s_config):
             tasks = cur.fetchall()
             con.close()
 
-            # task[1] is the todo_completed column
+            # task[0] is the task name, task[1] is the todo_completed column
             duetasks = [task[0] for task in tasks if task[1] is None]
             due = len(duetasks)
             completed = len(tasks) - due
             if due:
                 current = duetasks[0]
-
-            response = {
-                'cached_until': time() + self.cache_timeout,
-                'full_text': self.format.format(due=due,
-                                                completed=completed,
-                                                current=current)
-            }
-            return response
+            
+            return self._response(self.format.format(due=due,
+                                                     completed=completed,
+                                                     current=current)
```

Comment:
Thanks a lot for the help this is much better, there is just a missing closing parenthesis on L75. (beside that everything is working fine here)